software/spmd/gather_scatter_multi/Makefile

```diff
@@ -0,0 +1,20 @@
+bsg_tiles_X = 2
+bsg_tiles_Y = 2
+
+
+all: main.run
+
+OBJECT_FILES=main.o
+
+include ../Makefile.include
+
+# use 4x4_gs
+BSG_MACHINE_PATH = $(BSG_MANYCORE_DIR)/machines/4x4_gs
+
+main.riscv: $(OBJECT_FILES) $(SPMD_COMMON_OBJECTS) $(BSG_MANYCORE_LIB) crt.o
+	$(RISCV_LINK) $(OBJECT_FILES) $(SPMD_COMMON_OBJECTS) -L. -l:$(BSG_MANYCORE_LIB) -o $@ $(RISCV_LINK_OPTS)
```

Comment:
Can you change -l:$(BSG_MANYCORE_LIB) to "-l:$(BSG_MANYCORE_LIB)" in all instances?

Reply:
(This makes makefile syntax highlighting work in emacs - otherwise it breaks)

Reply:
Consider using vim.

Reply:
Thanks for the feedback!